src/main/java/xyz/nucleoid/plasmid/game/GameLogic.java

```diff
@@ -7,29 +7,41 @@
 import xyz.nucleoid.plasmid.game.rule.RuleResult;
 
 /**
- * Represents the logic of a game in a {@link GameWorld} through events ({@link EventListeners}) and rules ({@link GameRuleSet}).
+ * Represents the logic of a game in a {@link GameSpace} through events ({@link EventListeners}) and rules ({@link GameRuleSet}).
  *
- * <p>Each GameWorld contains one {@link Game} instance instance at a time.
- * Games can be swapped or opened through {@link GameWorld#setGame} or {@link GameWorld#openGame}.
+ * <p>Each GameSpace contains one {@link GameLogic} instance instance at a time.
+ * Games can be opened through {@link GameSpace#openGame}.
  */
-public final class Game {
+public final class GameLogic {
+    private final GameSpace space;
+
     private final EventListeners listeners = new EventListeners();
     private final GameRuleSet rules = new GameRuleSet();
+    private final GameResources resources = new GameResources();
 
-    public static Game empty() {
-        return new Game();
+    GameLogic(GameSpace space) {
+        this.space = space;
     }
 
-    public Game setRule(GameRule rule, RuleResult result) {
+    public GameLogic setRule(GameRule rule, RuleResult result) {
         this.rules.put(rule, result);
         return this;
     }
 
-    public <T> Game on(EventType<T> event, T listener) {
+    public <T> GameLogic on(EventType<T> event, T listener) {
         this.listeners.add(event, listener);
         return this;
     }
 
+    public GameLogic addResource(AutoCloseable resource) {
```

Comment:
Maybe a javadoc on this would be useful, I currently have no clue what is a resource in this context.